pkgs/development/libraries/ffmpeg/generic.nix

```
@@ -79,7 +78,7 @@
 , withSrt ? withHeadlessDeps # Secure Reliable Transport (SRT) protocol
 , withSsh ? withHeadlessDeps # SFTP protocol
 , withSvg ? withFullDeps # SVG protocol
-, withSvtav1 ? withFullDeps && !stdenv.isAarch64 # AV1 encoder/decoder (focused on speed and correctness)
+, withSvtav1 ? withSmallDeps && !stdenv.isAarch64 # AV1 encoder/decoder (focused on speed and correctness)
```

```suggestion
, withSvtav1 ? withHeadlessDeps && !stdenv.isAarch64 # AV1 encoder/decoder (focused on speed and correctness)
```

Headless systems also benefit from svt-av1 support. There's nothing "headed" about it.